### PR TITLE
Append [Datadog] to alerting logs

### PR DESCRIPTION
**Pull request by j-x-han**
# Description

See IDSEQ-2239.

As a temporary stopgap measure as we move away from using Datadog, relevant alerting log lines will have "[Datadog]" appended to the front, and only those logs will be sent via the Lambda. (The Lambda should probably be modified **after** this PR makes it into prod.)

# Notes

This should just be a temporary solution until the Datadog monitors are completely replaced.

### CODE REVIEW

File: app/jobs/autoscaling.py [hunk 1]
```diff
@@ -74,83 +74,90 @@ def autoscaling_update(config):
...
+                print "No instances are ready to move from 'draining' to 'discarded' state."
+
+        if 'batch_configurations' in config:
+            print "Autoscaling batches:"
+            response_batch_autoscaling = batch_autoscaling.autoscale_compute_environments(config['batch_configurations'])
+            print json.dumps(response_batch_autoscaling, default=_default_json_serializer)
+    except Exception as error:
+        # If any error occurs while performing an autoscaling update, append [Datadog] to ensure that the log is
+        # forwarded to Datadog.
+        # TODO(julie): Note that this is part of a temporary stopgap measure as we move away from Datadog and should be
```
Comment (jshoe) [inline app/jobs/autoscaling.py]: could you date this comment and also include the JIRA number?

Comment (jshoe) [reply app/jobs/autoscaling.py]: could you also `raise error` again at the end of this? in case someone ever adds more handling around this

**Comment by jshoe** [comment]
Thanks for taking this on and also volunteering to help follow through to remove DD logs entirely.

**Comment by jshoe** [comment]
There's also one more in idseq-dag for `AmrAlleleMismatchError` (rare) that you can change! Or maybe @MarkAZhang can comment on if that one is still necessary.

File: app/jobs/autoscaling.py (hunk 1 above)
Comment (j-x-han) [reply app/jobs/autoscaling.py]: Done!

**Comment by MarkAZhang** [comment]
If it's not too much trouble to keep it, I think it would be good. While it shouldn't get triggered under normal circumstances, if we were to ever update our AMR database, it might catch an error or two. I can still see it having use in the future.

**Comment by j-x-han** [comment]
I'll submit a PR to idseq-dag to update the `AmrAlleleMistmatchError` as well 😄

**Pull request merged**